### PR TITLE
Fix search popover sizing

### LINKED ISSUE #108: Location field is broken when user search any location in Map
### Describe the bug

When searching any location in Location Settings field then all suggested address by the system has a broken layout. 

### Steps to Reproduce

1. Open settings for Full site editing from Appearance > Editor
2. Add new Map Block
3. Search any address in 'Location Setting' field.
4. See broken results

### Screenshots, screen recording, code snippet

image

https://user-images.githubusercontent.com/6384299/151167354-14280036-6741-4dce-be8c-f97525d6d2c7.mp4

### Environment information

<Details>
`
### wp-core ###

version: 5.9
site_language: en_US
user_language: en_US
timezone: +00:00
permalink: undefined
https_status: false
multisite: false
user_registration: 0
blog_public: 1
default_comment_status: open
environment_type: local
user_count: 2
dotorg_communication: true

### wp-paths-sizes ###

wordpress_path: /Users/ank/Local Sites/miscwoo/app/public
wordpress_size: loading...
uploads_path: /Users/ank/Local Sites/miscwoo/app/public/wp-content/uploads
uploads_size: loading...
themes_path: /Users/ank/Local Sites/miscwoo/app/public/wp-content/themes
themes_size: loading...
plugins_path: /Users/ank/Local Sites/miscwoo/app/public/wp-content/plugins
plugins_size: loading...
database_size: loading...
total_size: loading...

### wp-active-theme ###

name: Twenty Twenty-Two (twentytwentytwo)
version: 1.0
author: the WordPress team
author_website: https://wordpress.org/
parent_theme: none
theme_features: core-block-patterns, post-thumbnails, responsive-embeds, editor-styles, html5, automatic-feed-links, block-templates, widgets-block-editor, wp-block-styles, editor-style
theme_path: /Users/ank/Local Sites/miscwoo/app/public/wp-content/themes/twentytwentytwo
auto_update: Disabled

### wp-themes-inactive (4) ###

Storefront: version: 3.9.1, author: Automattic, Auto-updates disabled
Twenty Nineteen: version: 2.1, author: the WordPress team (latest version: 2.2), Auto-updates disabled
Twenty Twenty: version: 1.8, author: the WordPress team (latest version: 1.9), Auto-updates disabled
Twenty Twenty-One: version: 1.4, author: the WordPress team (latest version: 1.5), Auto-updates disabled

### wp-plugins-active (3) ###

Block for Apple Maps: version: 1.0.1, author: 10up, Auto-updates disabled
WooCommerce: version: 6.1.0, author: Automattic (latest version: 6.1.1), Auto-updates disabled
WooCommerce Admin: version: 3.2.0-dev, author: WooCommerce, Auto-updates disabled

### wp-plugins-inactive (1) ###

WooCommerce PayFast Gateway: version: 1.4.19, author: WooCommerce (latest version: 1.4.20), Auto-updates disabled

### wp-media ###

image_editor: WP_Image_Editor_GD
imagick_module_version: Not available
imagemagick_version: Not available
imagick_version: Not available
file_uploads: File uploads is turned off
post_max_size: 1000M
upload_max_filesize: 300M
max_effective_size: 300 MB
max_file_uploads: 20
gd_version: bundled (2.1.0 compatible)
gd_formats: GIF, JPEG, PNG, BMP
ghostscript_version: not available

### wp-server ###

server_architecture: Darwin 21.1.0 x86_64
httpd_software: nginx/1.16.0
php_version: 8.0.0 64bit
php_sapi: fpm-fcgi
max_input_variables: 4000
time_limit: 1200
memory_limit: 256M
max_input_time: 600
upload_max_filesize: 300M
php_post_max_size: 1000M
curl_version: 7.54.0 LibreSSL/2.6.5
suhosin: false
imagick_availability: false
pretty_permalinks: true
htaccess_extra_rules: false

### wp-database ###

extension: mysqli
server_version: 5.7.28
client_version: mysqlnd 8.0.0
max_allowed_packet: 16777216
max_connections: 151

### wp-constants ###

WP_HOME: undefined
WP_SITEURL: undefined
WP_CONTENT_DIR: /Users/ank/Local Sites/miscwoo/app/public/wp-content
WP_PLUGIN_DIR: /Users/ank/Local Sites/miscwoo/app/public/wp-content/plugins
WP_MEMORY_LIMIT: 40M
WP_MAX_MEMORY_LIMIT: 256M
WP_DEBUG: true
WP_DEBUG_DISPLAY: true
WP_DEBUG_LOG: true
SCRIPT_DEBUG: true
WP_CACHE: false
CONCATENATE_SCRIPTS: undefined
COMPRESS_SCRIPTS: undefined
COMPRESS_CSS: undefined
WP_ENVIRONMENT_TYPE: local
DB_CHARSET: utf8
DB_COLLATE: undefined

### wp-filesystem ###

wordpress: writable
wp-content: writable
uploads: writable
plugins: writable
themes: writable

`
</Details> 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by fabiankaegy**
### Description of the Change

This is a quick fix for the width and overlap issues of the search popover. It also is a follow up to #99 since the new WordPress packages were released to NPM so this fixes our linting issues :) 

Closes #108 

### Checklist:

- [x] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests passed.

### Credits

Props @

### CODE REVIEW

File: src/components/SearchResults.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ export default function SearchResults(props) {
 
 	return (
 		<Popover focusOnMount={false}>
-			<ul>
+			<ul style={{ width: '250px' }}>
```
Comment (fabiankaegy) [inline src/components/SearchResults.js:12]: This right here is the actual bugfix. The rest is a cleanup for #99

Comment (dinhtungdu) [reply src/components/SearchResults.js:12]: @fabiankaegy Is that expected behavior for the `<Popover>` to be placed on the bottom left of the input?



image

Comment (fabiankaegy) [reply src/components/SearchResults.js:12]: @dinhtungdu I have not touched the anchoring of the popover for right now. This is how It was positioned before as far as I am aware. 

I don't have any more time to get this cleaned up more this week but I wanted to at least get the sizing & overlap bug fixed. 

I want to do an overhaul of the entire search component to be more visually pleasing and also more acessible. But that won't happen for this release.

Comment (dinhtungdu) [reply src/components/SearchResults.js:12]: I agree, just want to make sure you know about it. This PR is good to go now!

**Review by dinhtungdu — APPROVED** [review]
:shipit:

**Comment by ankitguptaindia** [comment]
Just reviewed with `develop` branch. Search suggestions look better now. Thanks for the fixes :) 



Screenshot 2022-01-27 at 7 58 12 PM

**Pull request merged**